pages/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+    <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Home | Grace Park</title>
+      <link href="../stylesheets/index.css" rel="stylesheet">
+    </head>
+
+    <body class="container">
+
+    <header><h1 class="header-title">Grace Park</h1></header>
+
+      <nav> 
+        <a href="index.html">Home</a> | 
+        <a href="about.html">About</a> |
+        <a href="portfolio.html">Portfolio</a>
+      </nav>
+      
+      <main>
+        <section>
+          <h1>Hi, I'm Grace.</h1>
+          <img id="gp" alt="Grace Park portfolio headshot - eadoyle" src="/assets/image0.jpg"/>
+          <p>I'm a Software Engineering Student at Ada Developers Academy based in the Greater Seattle Area.</p>
+          <p></p>
```

Comment:
Try to do spacing using CSS instead of empty paragraphs - it's more flexible and plays nicer with screen readers.